src/org/ods/component/ScanWithSnykStage.groovy

```diff
@@ -54,7 +60,7 @@ class ScanWithSnykStage extends Stage {
         script.error 'Snyk monitor failed'
       }
 
-      testSuccess = snyk.test(config.organisation, config.buildFile)
+      testSuccess = snyk.test(config.organisation, config.buildFile, config.severityThreshold)
```

Comment:
Just a question: Is this setting also applicable for the `monitor` command?

Reply:
I tried to find out but did not find any mentioning of this, I am also not sure if it makes sense considering the following description of it:

> snyk monitor can be used from the CLI to create a project on the Snyk website that will be continuously monitored for new vulnerabilities. After running this command you will see it by logging in to the website and viewing Your projects. You should run this command regularly in order to update the dependencies for your project. Consider running it as part of your build process after running snyk test, that way we will always know what dependencies you have and can notify you when any of them become vulnerable. 

Interestingly they write to do test before monitor, we do this currently the other way around, maybe we can swap that in another ticket.

Reply:
Oh good find. Could you create another issue for it?

Reply:
https://github.com/opendevstack/ods-jenkins-shared-library/issues/253